lispy.el

```diff
@@ -8579,23 +8584,18 @@ When ARG is non-nil, unquote the current string."
     (define-key map (kbd "M-?") 'lispy-convolute-sexp)
     (define-key map (kbd "M-S") 'lispy-split)
     (define-key map (kbd "M-J") 'lispy-join)
-    (define-key map (kbd "{") 'lispy-braces)
-    (define-key map (kbd "}") 'lispy-brackets)
-    (define-key map (kbd "]") 'lispy-forward)
-    (define-key map (kbd "[") 'lispy-backward)
-    (define-key map (kbd "M-(") 'evil-cp-wrap-next-round)
-    (define-key map (kbd "M-{") 'evil-cp-wrap-next-curly)
-    (define-key map (kbd "M-}") 'evil-cp-wrap-next-square)
-    (define-key map (kbd "<") 'evil-cp-<)
```

Comment:
Having <, > and y bound in insert mode (not just in special positions) isn't done by cleverparens and is undesirable in my opinion. It also shadows the binding for lispy-occur. I have added a separate evil-cp-special theme to make < and > behave like evil-cleverparens in special positions.